lib/rules/next-tick-style.js

```diff
@@ -79,17 +79,44 @@ function isAwaitedPromise(callExpression) {
   )
 }
 
+/**
+ * @param {CallExpression} callExpression
+ * @returns {boolean}
+ */
+
+/**
+ * @param {Expression | SpreadElement} callback
+ * @param {SourceCode} sourceCode
+ * @returns {string}
+ */
+function extractCallbackBody(callback, sourceCode) {
+  if (
+    callback.type !== 'FunctionExpression' &&
+    callback.type !== 'ArrowFunctionExpression'
+  ) {
+    return ''
+  }
+
+  if (callback.body.type === 'BlockStatement') {
+    return sourceCode.getText(callback.body).trim()
+  }
+
+  return sourceCode.getText(callback.body)
```

Comment:
The function body can't exact directly if it has `return`


```js
const foo = async () => {
	nextTick(() => {
		return;
	})

	return false
}
```

returns `false`


```js
const foo = async () => {
	await nextTick(); {
		return;
	}

	return false
}
```

returns `undefined`

Reply:
It will also broken if the callback has `.id` or used `arguments` inside.

```js
const foo = async () => { 
	nextTick(function foo() {
    use(foo)
	  use(arguments)
	})
}
```